airflow/plugins_manager.py

```diff
@@ -154,6 +156,8 @@ class AirflowPlugin:
     # buttons.
     operator_extra_links: List[Any] = []
 
+    custom_ti_deps: List[Any] = []
```

Comment:
The `custom_` prefix is I think implicit (it's a plugin, everything is custom), so we could just call this `ti_deps`